app/_src/documentation/requirements.md

```diff
@@ -10,11 +10,16 @@ This page expose the different requirements to run {{site.mesh_product_name}}.
 
 ## Kubernetes
 
-{{site.mesh_product_name}} is validated against Kubernetes `1.19.x`, `1.20.x`, `1.21.x` and `1.22.x`.
+{{site.mesh_product_name}} is validated against Kubernetes `1.20.x`, and `1.26.x`.
```

Comment:
How is this section different than https://github.com/kumahq/kuma-website/blob/51928e6e9ddf7c2131f9ecce148b78cde7094e95/app/_src/introduction/kuma-requirements.md?plain=1#L13 ?